src/main/java/bio/terra/cloudevents/v1/messagewrapper/EventMessage.java

```diff
@@ -0,0 +1,55 @@
+package bio.terra.cloudevents.v1.messagewrapper;
```

Comment:
I'd like to switch away from io.cloudevents for now.  I think it's premature to bring it into the mix.  Accordingly, I don't think this class is needed just yet.

Reply:
The `io.cloudevents` in [cloudevents/sdk-java](https://github.com/cloudevents/sdk-java/tree/master/core/src/main/java/io/cloudevents/core/v1) provides concrete implementation for the `CloudEvent` interface, it enables us to tell the event spec version and type, specified in [CloudEvents - Version 1.0](https://github.com/cloudevents/spec/blob/v1.0/spec.md) as REQUIRED attributes

- [specversion](https://github.com/cloudevents/spec/blob/v1.0/spec.md#specversion)
- [type](https://github.com/cloudevents/spec/blob/v1.0/spec.md#type)

In my opinion, using `CloudEventsFunction` and `CloudEvent` from [functions-framework-java](https://github.com/GoogleCloudPlatform/functions-framework-java) and [cloudevents/sdk-java](https://github.com/cloudevents/sdk-java) are the most robust way of parsing cloud events since we have a spec to follow.